addons/website/static/src/scss/primary_variables.scss

```diff
@@ -2083,6 +2084,8 @@ $o-website-values-palettes: (
         'btn-padding-y-lg': 1rem,
         'btn-padding-x-lg': 2.5rem,
         'btn-border-radius-lg': 2rem,
+
+        'input-background-color': #fff,
```

Comment:
Do you actually need to force to white in the default palette? Meaning only the default theme would have forced white inputs while all other themes would have body color enabled?